docs/core/install/linux-alpine.md

```diff
@@ -23,20 +23,11 @@ Installers aren't available for Alpine Linux. You must install .NET in one of th
 
```

Comment:
AFAIK, .NET is available in Alpine repositories starting with 3.15?

@ayakael any thoughts on how to update this installation doc?

Reply:
I had actually started working on updating it a few months ago, but lacked the time to complete it. I kind of just copied Fedora's page and updated it for Alpine. .NET 6 was initially included in 3.16, and .NET 7 is slated for 3.17.

Reply:
@omajid @ayakael does someone have instructions for the Alpine package manager? I'm not familiar with that distro in any way. Is it using a manager that we already have instructions for in the include section, such as yum or apt?

Reply:
I don't recall the exact install instructions, but to answer the yum/apt part: no, Alpine uses [`apk`](https://wiki.alpinelinux.org/wiki/Alpine_Package_Keeper).

Reply:
I tested this using the alpine containers.

```
$ docker pull alpine:3.17
$ docker run -it alpine:3.17
/ # apk add dotnet-sdk-7.0
fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/community/x86_64/APKINDEX.tar.gz
ERROR: unable to select packages:
  dotnet-sdk-7.0 (no such package):
    required by: world[dotnet-sdk-7.0]
/ # apk add dotnet-sdk-6.0
(1/17) Installing libgcc (12.2.1_git20220924-r4)
(2/17) Installing libstdc++ (12.2.1_git20220924-r4)
(3/17) Installing dotnet-host (6.0.11-r0)
(4/17) Installing dotnet6-hostfxr (6.0.11-r0)
(5/17) Installing icu-data-full (72.1-r1)
(6/17) Installing icu-libs (72.1-r1)
(7/17) Installing lttng-ust (2.13.3-r0)                                                                                                                       
(8/17) Installing xz-libs (5.2.8-r0)                                           
(9/17) Installing libunwind (1.6.2-r0)
(10/17) Installing dotnet6-runtime (6.0.11-r0)
(11/17) Installing aspnetcore6-runtime (6.0.11-r0)
(12/17) Installing aspnetcore6-targeting-pack (6.0.11-r0)
(13/17) Installing dotnet6-apphost-pack (6.0.11-r0)
(14/17) Installing dotnet6-targeting-pack (6.0.11-r0)
(15/17) Installing dotnet6-templates (6.0.111-r0)
(16/17) Installing netstandard21-targeting-pack (6.0.111-r0)
(17/17) Installing dotnet6-sdk (6.0.111-r0)         
Executing busybox-1.35.0-r27.trigger                                           
OK: 483 MiB in 32 packages
/ # dotnet --info
.NET SDK (reflecting any global.json):
 Version:   6.0.111
 Commit:    b3bb659a9d

Runtime Environment:
 OS Name:     alpine
 OS Version:  3.17
 OS Platform: Linux
 RID:         alpine.3.17-x64
 Base Path:   /usr/lib/dotnet/sdk/6.0.111/

global.json file:
  Not found

Host:
  Version:      6.0.11
  Architecture: x64
  Commit:       943474ca16

.NET SDKs installed:
  6.0.111 [/usr/lib/dotnet/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 6.0.11 [/usr/lib/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 6.0.11 [/usr/lib/dotnet/shared/Microsoft.NETCore.App]

Download .NET:
  https://aka.ms/dotnet-download

Learn about .NET Runtimes and SDKs:
  https://aka.ms/dotnet/runtimes-sdk-inf
```

Yeah, just an `apk add <package name>` is sufficient to install available versions of .NET.

Reply:
Indeed, that's the way. dotnet7 is still not merged but is on the way for 3.17 and will be in for main release for, at least, 'armv7`, `aarch64` and `x86_64`. By the way, the package is named `dotnetx-sdk` rather than `dotnet-sdk-x.x`. While we have a virtual package that points to the actual name, we diverged from the guidelines to match our own package naming guidelines.

I've got a working draft of the install instructions for Alpine, and will share when ready.

Reply:
@ayakael what do you mean by 3.16 doesn't support .NET 7 in the table?